front/lib/swr/mcp_servers.ts

```diff
@@ -1054,7 +1054,7 @@ function useMCPServerViewsFromSpacesBase(
 
   return {
     serverViews: data?.serverViews ?? emptyArray(),
-    isLoading: !error && !data,
+    isLoading: !error && !data && spaces.length !== 0,
```

Comment:
I removed it in my previous PR but actually you need it, if there is no spaces, it should not be loading